src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -176,6 +183,15 @@ private void CheckModifiers(MethodKind methodKind, bool hasBody, Location locati
             {
                 diagnostics.Add(ErrorCode.ERR_ConstructorInStaticClass, location);
             }
+            else if (IsPartial && !ContainingType.IsPartial())
+            {
+                diagnostics.Add(ErrorCode.ERR_PartialMemberOnlyInPartialClass, location);
+            }
+
+            if (IsPartial)
+            {
+                ModifierUtils.CheckFeatureAvailabilityForPartialEventsAndConstructors(location, diagnostics);
```

Comment:
>CheckFeatureAvailabilityForPartialEventsAndConstructors [](http://example.com/codeflow?start=30&length=55)

Does this check result in any diagnostics? When compiling with earlier language versions, I thought we treated `partial` as a type name for a regular method rather than as a constructor modifier.

Reply:
Good catch, that's outdated code, I will remove it, thanks.